lib/datastore/request.js

```diff
@@ -432,19 +460,36 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  *   // entities[].data = Empty object
  * });
  */
-DatastoreRequest.prototype.runQuery = function(query, callback) {
+DatastoreRequest.prototype.runQuery = function(query, options, callback) {
   var self = this;
 
+  if (is.fn(options)) {
+    callback = options;
+    options = {};
+  }
+
+  options = options || {};
+
   var protoOpts = {
     service: 'Datastore',
     method: 'runQuery'
   };
 
   var reqOpts = {
-    readOptions: {},
     query: entity.queryToQueryProto(query)
   };
 
+  if (options.consistency) {
+    var consistencyMap = {
```

Comment:
Looks like this map/logic is duplicated. Could we just make the `consistencyMap` global to this file? Or put it in a static(ish) method that retrieved the proper value?